### ISSUE TITLE
TypeError: Cannot read property 'length' of null

### ISSUE THREAD

**Issue by gauravarora** [fraywing/textAngular#987]
I am not sure what I am doing wrong but I always see this exception in my console even though textAngular continues to work just fine. I'd like to understand what I am doing wrong and remove the exception.

```
TypeError: Cannot read property 'length' of null
    at l.taRegisterTool.activeState (http://cdnjs.cloudflare.com/ajax/libs/textAngular/1.4.6/textAngularSetup.js:806:25)
    at http://cdnjs.cloudflare.com/ajax/libs/textAngular/1.4.6/textAngular.js:2674:39
    at Object.s [as forEach] (http://cdnjs.cloudflare.com/ajax/libs/angular.js/1.3.7/angular.min.js:7:408)
    at http://cdnjs.cloudflare.com/ajax/libs/textAngular/1.4.6/textAngular.js:2671:16
    at Object.s [as forEach] (http://cdnjs.cloudflare.com/ajax/libs/angular.js/1.3.7/angular.min.js:7:302)
    at Object.editors.(anonymous function).editorFunctions.updateSelectedStyles (http://cdnjs.cloudflare.com/ajax/libs/textAngular/1.4.6/textAngular.js:2670:15)
    at l.scope.updateSelectedStyles (http://cdnjs.cloudflare.com/ajax/libs/textAngular/1.4.6/textAngular.js:2554:17)
    at http://cdnjs.cloudflare.com/ajax/libs/textAngular/1.4.6/textAngular.js:2605:13
    at l.$eval (http://cdnjs.cloudflare.com/ajax/libs/angular.js/1.3.7/angular.min.js:126:84)
    at l.$apply (http://cdnjs.cloudflare.com/ajax/libs/angular.js/1.3.7/angular.min.js:126:310)
```
I am using textAngular version 1.4.6, Angular 1.3.7 and am on latest stable build of Chrome.

My HTML is:
```
<text-angular ng-model="form.terms" name="terms_editor" ta-text-editor-class="form-control myform1-height" ta-html-editor-class="form-control myform1-height">
</text-angular>
```

**Comment by cocojack42** [comment]
I have the same problem, I think it's because of the wordcount in the toolbar.

**Comment by gauravarora** [comment]
Is there anyway to disable wordcount?

**Comment by cocojack42** [comment]
Sure, just define the toolbar without the wordcount

https://github.com/fraywing/textAngular#customization

**Comment by jshamley** [comment]
FYI - initializing the variable with a space did not fix the error for me. I had to initialize the variable with `$scope.content = new String();`.

**Comment by JoelParke** [comment]
This is fixed with PR #1321.  and will be in the next release.
